### ISSUE TITLE
Voting

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/organization#10]
We'll be voting on our logo soon, and because consensus can be such a tricky thing in such a large group of people, we should probably do a vote. How do we handle voting? 

I was hoping to write up a blog post discussing all of these things, but I just haven't had the time.

Here's some reading material:
* [Through Hell and High Water](http://www.akpress.org/comehellorhighwater.html) a book about community discussions etc
* [The Tyranny of Structurelessness](http://www.jofreeman.com/joreen/tyranny.htm)
* [The Tyranny of Tyranny](http://libcom.org/library/tyranny-of-tyranny-cathy-levine) Response to the above essay
* [The Theology of Consensus](https://www.jacobinmag.com/2015/05/consensus-occupy-wall-street-general-assembly/)
* [Consensus is a means, not an end](http://beautifultrouble.org/principle/consensus-is-a-means-not-an-end/)
* [Powershift Guidelines](https://docs.google.com/document/d/1g2QbdXgHNQPDTsrqrhT6Kuvv2znUTE9Qn6YxxnWEny0/edit) just a good example of guidelines, check the Skype conversation bit
* [Robert's Rules](https://en.wikipedia.org/wiki/Robert%27s_Rules_of_Order)

(cc @GarthDB @bnvk before you hi-jack the logo thread :P)

**Comment by GarthDB** [comment]
@simonv3 you're so smart, I was about to hijack the logo issue.

Voting for a logo sounds a lot like design by committee: in the research we've done it's one of the things that scares people (particularly designers) away from contributing open source.

Ultimately the logo should be decided by the core contributor in the creative director role with the advice/council of the core team. In the case of Open Source Design we have 150 people on the team, that makes it hard to figure out who the owner of this decision is.

Let me know if I'm completely off base with my thinking.

**Comment by GarthDB** [comment]
With no owner, voting is really our only option.

**Comment by simonv3** [comment]
@GarthDB Yeah, the design by committee thing is what I'm most hesitant about - but I'm also down for exploring a space where designing in the open, with lots of feedback and lots of ideas coming in from all sorts of angles, has the best result. So far I think the back and forth in that thread has had really positive results, though I'm not sure how other people feel about that. Have you put together the research into a blog post yet of some kind? Would love to see that.

I think there's a couple of voting directions we can go in. I suspect that what @bnvk is suggesting with a ranking of each logo would end us with the least offensive logo, rather than the best logo, but maybe not. Maybe just a straight vote - "which one do you think is best" would work, without emphasis on ranking them all? 

Maybe we don't even vote on the logo, but vote on someone who should make the decision (being designated as the lead) or a small group of people ("experts" and "non-experts") to come to a consensus on the logo (rather than 150 people throwing in their opinion). There's definitely a sentiment in open source, as well as design, that a "benevolent dictator" gets us places. I would prefer to challenge that, but am happy to pick one battle at a time.

I think that with a group this size, and one that's (hopefully) going to grow, we're going to need some sort of structure. To me that would be preferably non-hierarchical, but I'm not opposed to little sub groups (call the committees), but I'm open to discussion.

**Comment by davelab6** [comment]
I recommend asking someone to step up to the creative director role.

**Comment by gniezen** [comment]
Wouldn't "design by committee" mean that the votes are able to influence the design of the logo? I don't see anything wrong with the most popular logo being chosen through votes, as long as the votes don't influence the actual design of the logo.

To me, voting to select a committee with a creative director who gets to decide what logo is used sounds *exactly* like design by committee.

**Comment by Xaviju** [comment]
I don't like voting, and I don't like comitees because its kind of imposition (yes, they are useful if there are no other options) and structural (again, avoid if not necessary)
I like the idea of improving the logo in different ways from a parent idea as it happened in the thread. But I understand that consensus is somenthing hard to accomplish here.

My proposal is to set a online event to discuss current ideas and try to find a consensus for this important matter and iterate again. Im sure communication makes in 1h more than Github issues in 5 weeks.

**Comment by simonv3** [comment]
@bnvk Heh, thanks I was just about to change the title to something similar :)

**Comment by simonv3** [comment]
How do people feel about having a chat Sunday the 28th of June about how we're going to decide on the logo / deciding on the logo? @GarthDB @jancborchardt @Xaviju @gniezen @davelab6 @bnvk & anyone else. If that doesn't work for everyone, how about July 5th?

**Comment by gniezen** [comment]
@simonv3  I totally agree that our brand identity should have a broad base of support. Wouldn't a simple vote for most popular logo achieve that? We don't want to end up with a committee with the power to tell a designer how to improve the logo ("we want to use this one, but can you make it pop?" :stuck_out_tongue_closed_eyes: ) 

I'm a bit hesitant of selecting a creative director as that starts to impose hierarchy, and you're either lucky to have a great person that makes good decisions, or someone that makes decisions that ends up alienating the whole community. Do people in this community know each other well enough to be able to select a good creative director?

If we're having a meeting, what's the suggested UTC time? (I'm UTC+1)

**Comment by Xaviju** [comment]
@simonv3 Ill be happy to discuss what's the best way to achieve consensus (not only for the logo, but for everything else). I'm not sure if I'll be available on sunday but I trust you on this decission. To me, July 5th would be better.

**Comment by jancborchardt** [comment]
July 5th works better for me as well.

**Comment by simonv3** [comment]
@jancborchardt That would be good. I was going to put a page together of all the things from the thread that would hopefully update automatically if someone puts something new in there (playing with the Github API). I'll also number renditions, and then we can create a typekit form that lets people choose an option?

**Comment by Xaviju** [comment]
Sure, there is also a nice tools for group decission (I used it already in the past) called Loomio. https://www.loomio.org/
Might be overengineering but I love how simple is to organize group decissions.

**Comment by simonv3** [comment]
I've thought about suggesting loomio for this community in the past, I'm
just hesitant to bring on board another tool. My experience with loomio is
that everyone is super excited about it in the beginning, but then that
dies out and things move back to github.

**Comment by jancborchardt** [comment]
Loomio just puts way too much emphasis on quantitative voting, and it’s yet another platform. I think we can pretty well talk decisions like that through. :)

There’s also reasoning beyond votes, for example my opinion on the logo has changed a great deal through the collaborative design process. Would be good if we can finalize that in a mentioned call.

**Comment by Xaviju** [comment]
:+1:

**Comment by simonv3** [comment]
So new date proposed - July 5th. Time preference? 

One thought: 10AM PDT which is 7 PM UTC + 2 (Germany).

**Comment by jancborchardt** [comment]
Sounds good.

**Comment by una** [comment]
for reference: http://everytimezone.com/#2015-6-28,180,cn3

I think it would be nice to use everytimezone.com whenever a date is mentioned as an idea for collaboration docs moving forward.

And yes, I can do that time

**Comment by gniezen** [comment]
:+1:

**Comment by simonv3** [comment]
I completely blanked on this - did this happen?

**Comment by jancborchardt** [comment]
Not that I know of – @bnvk and me also missed it. European heatwave … ;D

I think anyway that we can’t decide from what we currently have. We should whittle down the logo choices and then do another iteration from there maybe.

**Comment by jancborchardt** [comment]
(Also clearly defining what we want it to convey.)

**Comment by simonv3** [comment]
can you elaborate a bit more with what you think some good steps are?
